apps/glusterfs/volume_entry.go

```diff
@@ -219,6 +223,20 @@ func (v *VolumeEntry) Unmarshal(buffer []byte) error {
 	return nil
 }
 
+// HasArbiterOption returns true if this volume is flagged for
+// arbiter support.
+func (v *VolumeEntry) HasArbiterOption() bool {
+	for _, s := range v.GlusterVolumeOptions {
+		r := strings.Split(s, " ")
```

Comment:
I thought glusterVolumeOptions are comma separated?

Reply:
On the command line, between k-v pairs, sure. But from what I could tell in the code and the existing examples the key and the value are separated by a space. A couple examples:
```
apps/glusterfs/block_volume_entry.go:   msg.GlusterVolumeOptions = []string{"group gluster-block"}
tests/functional/TestSmokeTest/tests/heketi_test.go:    volReq.GlusterVolumeOptions = []string{"performance.rda-cache-limit 10MB"}
```
From what I can tell the strings are passed to the gluster command line as is and gluster expects something like "gluster --mode=script volume set VOLNAME KEY VALUE".

Reply:
You are right! I did not check properly.